src/__tests__/patients/GeneralInformation.test.tsx

```diff
@@ -21,95 +23,76 @@ describe('General Information', () => {
     address: 'address',
     friendlyId: 'P00001',
     dateOfBirth: new Date().toISOString(),
+    isApproximateDateOfBirth: false,
   } as Patient
 
-  const setup = () => {
-    const wrapper = mount(<GeneralInformation patient={patient} />)
+  let wrapper: ReactWrapper
+  let history = createMemoryHistory()
 
-    wrapper.update()
-    return wrapper
-  }
+  beforeEach(() => {
+    history = createMemoryHistory()
+    wrapper = mount(
+      <Router history={history}>
+        <GeneralInformation patient={patient} />)
+      </Router>,
+    )
+  })
 
   beforeEach(() => {
     jest.restoreAllMocks()
   })
 
   it('should render the sex select', () => {
-    const wrapper = setup()
-
     const sexSelect = wrapper.findWhere((w: any) => w.prop('name') === 'sex')
     expect(sexSelect.prop('value')).toEqual(patient.sex)
     expect(sexSelect.prop('label')).toEqual('patient.sex')
     expect(sexSelect.prop('isEditable')).toBeFalsy()
   })
 
   it('should render the patient type select', () => {
-    const wrapper = setup()
-
     const typeSelect = wrapper.findWhere((w: any) => w.prop('name') === 'type')
     expect(typeSelect.prop('value')).toEqual(patient.type)
     expect(typeSelect.prop('label')).toEqual('patient.type')
     expect(typeSelect.prop('isEditable')).toBeFalsy()
   })
 
-  it('should render the age of the patient', () => {
-    const wrapper = setup()
-
-    const ageInput = wrapper.findWhere((w) => w.prop('name') === 'age')
-    expect(ageInput.prop('value')).toEqual('0')
-    expect(ageInput.prop('label')).toEqual('patient.age')
-    expect(ageInput.prop('isEditable')).toBeFalsy()
-  })
-
   it('should render the date of the birth of the patient', () => {
-    const wrapper = setup()
-
-    const dateOfBirthInput = wrapper.findWhere((w) => w.prop('name') === 'dateOfBirth')
+    const dateOfBirthInput = wrapper.findWhere((w: any) => w.prop('name') === 'dateOfBirth')
     expect(dateOfBirthInput.prop('value')).toEqual(new Date(patient.dateOfBirth))
     expect(dateOfBirthInput.prop('label')).toEqual('patient.dateOfBirth')
     expect(dateOfBirthInput.prop('isEditable')).toBeFalsy()
   })
 
   it('should render the occupation of the patient', () => {
-    const wrapper = setup()
-
-    const dateOfBirthInput = wrapper.findWhere((w) => w.prop('name') === 'occupation')
+    const dateOfBirthInput = wrapper.findWhere((w: any) => w.prop('name') === 'occupation')
```

Comment:
this should probably be `occupationInput`

Reply:
done